ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -127,6 +127,9 @@ public int execute() {
       if (shouldLoadAtlasMetadata()) {
         addAtlasLoadTask();
       }
+      if (conf.getBoolVar(HiveConf.ConfVars.REPL_RANGER_HANDLE_DENY_POLICY_TARGET)) {
```

Comment:
if this conf is set to false but ranger replication is enabled what do you do

Reply:
ranger deny task will not be initiated.